### DISCUSSION TITLE
Cannot publish public template

### DISCUSSION THREAD

**Discussion by Lory1990** [mia-platform/community#414] [Q&A] [answered]
### What product are you having troubles with?

Marketplace

### What Console version are you using?

latest

### Description

when i add
```
"visibility": {
    "allTenants": true
}
```

to my configuration miactl shows the following message:
```
"visibility" field is not expected to be defined  
```

**Comment by pmarchini** [comment, marked as answer]
Ciao Lorenzo,

Thanks for reaching out!

At the moment the only way to set a marketplace item visibility is via CMS.
I see that our documentation is not clear enough regarding this specific topic.

We're going to fix it as soon as possible.

In the meantime, in order to publish a marketplace item, you can follow this [small guide](https://docs.mia-platform.eu/docs/11.x/marketplace/add_to_marketplace/marketplace_items_accessibility#open-an-issue-on-mia-platform-github-community-page).

Let me know if you have any questions or need further assistance!

Ciao!
